packages/webapp/src/views/developer/consent-agreement-upload.html

```diff
@@ -0,0 +1,48 @@
+{% extends 'multipart-form-layout.html' %}
+
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/file-upload/macro.njk" import govukFileUpload %}
+
+{% set pageHeading = "Upload written consent to use a biodiversity gain site for off-site gain" %}
+
+{% block formContent %}
+  <div class="govuk-body">
+    {% if err %}
+      {% include "errorbanner.html" %}
+    {% endif %}
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-l">
+          {{ pageHeading }}
+        </h1>
+
+        <div id="written-consent-hint">
+          <p>This needs to include:</p>
+          <ul class="govuk-list govuk-list--bullet">
+            <li>a written statement confirming the landowners' consent</li>
+            <li>the landowner's signature</li>
```

Comment:
Inconsistent use of apostrophe - landowners' and landowner's. Although the text is taken from the prototype, I suggest the latter is correct.

Reply:
I agree and Removed the apostrophe from the text though its there in prototype